lms/templates/courseware/course_navigation.html

```diff
@@ -86,21 +86,11 @@
     <ol class="course-tabs">
       % for tab in get_course_tab_list(request, course):
         <%
-            tab_is_active = (tab.tab_id == active_page) or (tab.tab_id == default_tab)
+        tab_is_active = (tab.tab_id == active_page) or (tab.tab_id == default_tab)
```

Comment:
ok, I'll change this string. But this behavior [was before](https://github.com/edx/edx-platform/blob/master/lms/templates/courseware/course_navigation.html#L89) my pull request :-)